htconfig.php

```diff
@@ -24,11 +24,15 @@
 // Use environment variables for mysql if they are set beforehand
 if (!empty(getenv('MYSQL_HOST'))
 	&& !empty(getenv('MYSQL_PORT'))
-	&& !empty(getenv('MYSQL_USERNAME'))
+	&& (!empty(getenv('MYSQL_USERNAME')) || !empty(getenv('MYSQL_USER')))
```

Comment:
Why do we have to support both?

Reply:
Mariadb => MYSQL_USER
MySQL => MYSQL_USERNAME

Reply:
Mariadb => MYSQL_USER
MySQL => MYSQL_USERNAME

Reply:
I know this is a while back already, but why exactly does MariaDB need a different environment variable?
From the code perspective, it seems like the same thing is happening for both anyway.

Referring to this: https://github.com/friendica/friendica/blob/71a60c5f66bd1f3f71e908241db711f5e4e1088d/src/App.php#L578-L582

Reply:
Yeah, we added support for both on @nupplaphil suggestion.

Reply:
I understand the idea behind it, but my question is rather: Why are there separate variables for MariaDB and MySQL?

Looking at the code, it doesn't really matter which one is set, as the same thing happens either way.

Or is there something else reading the environment variables that I'm not aware of?

Reply:
It seems specific to Docker, the main MySQL image expects `MYSQL_USERNAME` while the main MariaDB image expects `MYSQL_USER`. 🤷‍♂️

Reply:
Sorry for my real late answer, I totally overlooked it..

This was Docker specific.
In the meantime, they both support `MYSQL_USER`. So maybe we can remove it again I guess.